node/src/Pos/Worker.hs

```diff
@@ -80,6 +80,6 @@ allWorkers NodeResources {..} = mconcatPair
   where
     NodeContext {..} = nrContext
     properSlottingWorkers =
-       fst (localWorker (recoveryCommGuard "logNewSlot" logNewSlotWorker)) :
+       fst (localWorker logNewSlotWorker) :
```

Comment:
Explain this change please.

Reply:
I think explanation in the commit message is good enoguh: https://github.com/input-output-hk/cardano-sl/pull/1669/commits/7a9041ac91cbaf8373ab39bbe546a15ed6af5aae